extensions/2.0/Khronos/KHR_node_visibility/README.md

```diff
@@ -0,0 +1,63 @@
+# KHR_node_visibility
```

Comment:
Nitpick: We often escape underscores when outside of code blocks, such as this title string here.  In markdown, underscores are used to denote _italics_, so the above string may be rendered by some systems with _node_ in italics.  At some point GitHub markdown became smart enough to avoid mid-word italics like this, so the problem isn't always visible anymore.  But for future-proofing and compatibility with other markdown systems, I still recommend following what we've done in other extensions:

`# KHR\_node\_visibility`